spec/inverted-index-test.js

```diff
@@ -20,13 +21,13 @@ describe('Inverted Index Suite', () => {
 
   describe('Tokenize String Method', () => {
```

Comment:
Why the use of `Method` in your description?

I think it's redundant because you're running unit tests for methods already.

Kindly address this in all areas where they're used.